server/channels/app/team_test.go

```diff
@@ -646,23 +685,24 @@ func TestPermanentDeleteTeam(t *testing.T) {
 		Method:    model.CommandMethodPost,
 	})
 	require.Nil(t, err, "Should create a command")
-	defer th.App.DeleteCommand(command.Id)
+	defer func() {
+		appErr := th.App.DeleteCommand(command.Id)
+		require.Nil(t, appErr)
+
+		command, appErr = th.App.GetCommand(command.Id)
+		require.Nil(t, command, "command was deleted")
+		require.NotNil(t, appErr, "should return an error")
+	}()
 
 	command, err = th.App.GetCommand(command.Id)
 	require.NotNil(t, command, "command should not be nil")
 	require.Nil(t, err, "unable to get new command")
 
-	err = th.App.PermanentDeleteTeam(th.Context, team)
-	require.Nil(t, err)
-
-	command, err = th.App.GetCommand(command.Id)
-	require.Nil(t, command, "command wasn't deleted")
-	require.NotNil(t, err, "should not return an error")
```

Comment:
We still need that check. It confirms that after deleting a test, the command doesn't exist any longer.

Reply:
When I removed the `PermanentDeleteTeam`, I saw that there were 2 `GetCommand`s following each other so it didn't really make sense, so I moved it into the `DeleteCommand` defer but I will move it back if that's preferable. 
Do you want the `PermanentDeleteTeam` back too?